modules/olm-deleting-operators-from-a-cluster-using-web-console.adoc

```diff
@@ -23,13 +23,15 @@ endif::[]
 
 . On the right-hand side of the *Operator Details* page, select *Uninstall Operator* from the *Actions* drop-down menu.
 +
-An *Uninstall Operator?* dialog box is displayed, reminding you that:
+An *Uninstall Operator?* dialog box is displayed, alerting you that:
 +
 [.small]
 --
-*Removing the Operator will not remove any of its custom resource definitions or managed resources. If your Operator has deployed applications on the cluster or configured off-cluster resources, these will continue to run and need to be cleaned up manually.*
+*Operator VerticalPodAutoscaler will be removed from openshift-vertical-pod-autoscaler. Click on the checkbox below to also remove all Operands associated with this Operator. If your Operator configured off-cluster resources, these will continue to run and require manual cleanup.*
```

Comment:
This isn't necessarily the case, assuming that finalizers were properly created on crs by the operator and they properly wrote cleanup logic. It may make sense to call out that they *may* require manual cleanup, rather than they will always require manual cleanup.

Reply:
This is the message that pops up in console's dialog box while trying to delete an operator. Probably a good input for the console team to correct the modifications.
@kdoberst - This could be a discussion point for you may be?

Reply:
@kevinrizza @snarayan-redhat 
Just so I understand, we are talking about changing the text to:

... If your Operator deployed applications on the cluster or configured off-cluster resources, these will continue to run and may require manual cleanup.

Reply:
@kdoberst - Since the 4.9 version message is different as shown in green above, it would be this:  _If your Operator configured off-cluster resources, these will continue to run and **may** require manual cleanup._

Reply:
+1